common-service-core/src/main/java/org/zowe/apiml/product/version/BuildInfo.java

```diff
@@ -29,6 +32,7 @@ public BuildInfo(String buildProperties, String gitProperties) {
         this.gitProperties = gitProperties;
     }
 
+    @Autowired
```

Comment:
```suggestion
```

Nothing to be autowired. Actually, the second constructor could be removed, because it is used just in the test.

Reply:
This is to address a sonarqube issue, nothing more